app/src/main/java/io/homeassistant/companion/android/sensors/NextAlarmReceiver.kt

```diff
@@ -0,0 +1,39 @@
+package io.homeassistant.companion.android.sensors
+
+import android.app.AlarmManager
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import io.homeassistant.companion.android.common.dagger.GraphComponentAccessor
+import io.homeassistant.companion.android.domain.integration.IntegrationUseCase
+import javax.inject.Inject
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.Job
+import kotlinx.coroutines.launch
+
+class NextAlarmReceiver() : BroadcastReceiver() {
+
+    @Inject
+    lateinit var integrationUseCase: IntegrationUseCase
+    private val ioScope = CoroutineScope(Dispatchers.IO)
+    var updateJob: Job? = null
+    override fun onReceive(context: Context, intent: Intent) {
+        val isBootIntent = Intent.ACTION_BOOT_COMPLETED.equals(intent.action, ignoreCase = true)
+        val isNextAlarmIntent = AlarmManager.ACTION_NEXT_ALARM_CLOCK_CHANGED.equals(intent.action, ignoreCase = true)
+            if (!isBootIntent && !isNextAlarmIntent) {
+                return
+            }

```

Comment:
Is it just me or is this spacing a little off?

Reply:
I just manually checked and they seemed to be fine.  Linter complained about one of the spaces being 5 instead of 8 so I would hope it would catch this too :)

Reply:
Tabs vs spaces maybe?

Reply:
Just checked, there are no tabs in this method.  Also cross checked in notepad++ that uses dots or arrows for spaces and tabs

![image](https://user-images.githubusercontent.com/1634145/89802868-ece53900-dae6-11ea-87cd-309a128e0df4.png)

Reply:
In android studio go to File > Settings > Editor > General > Appearance and turn on "show white spaces" it will do the same thing :)

Reply:
Yup you were right, too many indents 🤦 lol